tests/suites/test_suite_pkcs1_v15.function

```diff
@@ -77,6 +80,9 @@ void pkcs1_rsaes_v15_decrypt(int mod, char *input_P, char *input_Q,
     TEST_ASSERT(mbedtls_rsa_set_padding(&ctx,
                                         MBEDTLS_RSA_PKCS_V15, hash) == 0);
 
+    TEST_ASSERT(mbedtls_rsa_get_padding_mode(&ctx) == MBEDTLS_RSA_PKCS_V15);
```

Comment:
Why did you change `TEST_EQUAL` to `TEST_ASSERT`? `TEST_EQUAL` is better: we get better feedback if the test fails.

Reply:
I'm bit of unclear the difference between `TEST_EQUAL` and `TEST_ASSERT`. `TEST_EQUAL` is used to check integer but requires signedness. But `MBEDTLS_RSA_PKCS_V15` and `MBEDTLS_RSA_PKCS_V21` are defined through #define Directive so I think `TEST_ASSERT` should be a better option. So I made this change.

Reply:
`TEST_EQUAL` works with any integral values including enums. It doesn't care about signedness (the two arguments should have the same signedness, but anyway, if you're comparing two values, they should typically have the same type).

Reply:
Understood, thanks. I'll change it.